packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -345,6 +361,17 @@ export class DatabaseCluster extends DatabaseClusterBase {
 
     // Get the actual subnet objects so we can depend on internet connectivity.
     const internetConnected = props.instanceProps.vpc.selectSubnets(props.instanceProps.vpcSubnets).internetConnectivityEstablished;
+
+    let monitoringRole;
+    if (props.monitoringInterval && props.monitoringInterval.toSeconds()) {
+      monitoringRole = props.monitoringRole || new Role(this, "MonitoringRole", {
```

Comment:
It would be nice to have the same optional `monitoringRole` for an instance.

Reply:
ok! done